dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/engine/command/impl/MoveDataverseCommand.java

```diff
@@ -134,7 +133,7 @@ public void executeImpl(CommandContext ctxt)  {
 
         // generate a list of templates in destination to check against
         List<Template> destinationTemplates = null;
-        if (moved.getTemplates() != null) {
+        if (!moved.getTemplates().isEmpty()) {
```

Comment:
Are we sure this won't ever be null at this point? If not, then I suggest usage of something that makes null check first eg. `if(CollectionUtils.isNotEmpty(temps))`

Reply:
I added default list to Entity so it should never be null, I also checked getters/setter just in case and I don't see any null set. But yeah I can also go for defensive programming here.